docs/conf.py

```diff
@@ -117,7 +117,9 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+html_theme_options = {
```

Comment:
@rosatamsen  I think the codecov drop is fine, because you uncommented this line.